File: Python/BubbleSort/bubble_sort.py
# Bubble sort, swaps adjacent element if they are in the wrong order

def bubbleSort(list):
  for i in range(0, len(list)):
    for j in range(0, len(list) - 1):
      if list[j] > list[j + 1]:
        list[j], list[j + 1] = list[j + 1], list[j]
  print(i,j)
  return list
result = bubbleSort([1,5,6,8,3,6])
print(result)

File: JavaScript/BubbleSort/bubbleSort.js
// Bubble sort, swaps adjacent element if they are in the wrong order

const bubbleSort = arr => {
  for (let i = 0; i < arr.length; i++) {
    for (let j = 0; j < arr.length - 1; j++) {
      if (arr[j] > arr[j + 1]) {
        let temp = arr[j];
        arr[j] = arr[j + 1];
        arr[j + 1] = temp;
      }
    }
  }
  return arr;
}

const result = bubbleSort([4,2,5,6,3,9,1,8,7]);
console.log(result);